frontend/public/components/modals/configure-machine-autoscaler-modal.tsx

```diff
@@ -56,16 +69,20 @@ const ConfigureMachineAutoscalerModal = (props: ConfigureMachineAutoscalerModalP
       },
     };
     return k8sCreate(MachineAutoscalerModel, machineAutoscaler);
```

Comment:
we may use non-deprecated `k8sCreateResource` here
```suggestion
    return k8sCreateResource({ model: MachineAutoscalerModel, data: machineAutoscaler });
```